### ISSUE TITLE
assert.Equal values printed as if they were format strings

### ISSUE THREAD

**Issue by asf-stripe** [stretchr/testify#484]
Running this test code:

``` go
package main

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestInequality(t *testing.T) {
	assert.Equal(t, "something%2Ffoo", "something-else%2Ffoo", "An example inequality with url-escapes")
}
```

I get the following output:

```
go test -run='TestInequality' .
--- FAIL: TestInequality (0.00s)
	Error Trace:	testify_test.go:10
	Error:      	Not equal: 
	            	expected: "something%!F(MISSING)foo"
	            	received: "something-else%!F(MISSING)foo"
	Messages:   	An example inequality with url-escapes
FAIL
```

I would expect the value to be printed literally, not as a Printf-like format strings with missing args.

**Comment by ikdc-stripe** [comment]
Looks like a duplicate of #325 which was fixed by 09f61d78b810439bdcfe332561533990811b1b92.

**Comment by asf-stripe** [comment]
Yep, turns out this was solved and my gopath seems to have gotten messed up. Closing.
